lego/apps/users/migrations/0040_rename_activation_date_penalty_activation_time.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.0.10 on 2023-04-26 13:39
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('users', '0039_penalty_activation_date'),
+    ]
+
+    operations = [
```

Comment:
This migration should be squashed with the previous one

Reply:
Shall do